src/datanode/src/sql/create.rs

```diff
@@ -249,16 +251,32 @@ impl SqlHandler {
             region_numbers: vec![0],
             primary_key_indices: primary_keys,
             create_if_not_exists: stmt.if_not_exists,
-            table_options: HashMap::new(),
+            table_options,
         };
         Ok(request)
     }
 }
 
+fn stmt_options_to_hashmap(opts: &[SqlOption]) -> error::Result<TableOptions> {
```

Comment:
Seems `TableOptions` is not a hashmap?

```suggestion
fn as_table_options(opts: &[SqlOption]) -> error::Result<TableOptions> {
```